sys/decl.go

```diff
@@ -110,22 +111,17 @@ const (
 
 type BufferType struct {
 	TypeCommon
-	Kind BufferKind
+	Kind       BufferKind
+	RangeBegin uintptr // for BufferBlobRange kind
+	RangeEnd   uintptr // for BufferBlobRange kind
 }
 
 func (t BufferType) Size() uintptr {
-	switch t.Kind {
-	case BufferAlgType:
-		return 14
-	case BufferAlgName:
-		return 64
-	default:
-		panic(fmt.Sprintf("buffer size is not statically known: %v", t.Name()))
-	}
+	panic("not called")
```

Comment:
I've restored struct alignment and made it possible to add tests for descriptions and serialization of different features. See 3ca39dfc4dd8a2dda5b3a9b8c29e295839490af3